week-2-practise/level-2/1-boolean.js

```diff
@@ -3,11 +3,12 @@
 // Fix it.
 
 var isHappy = "false";
+var isHappy = "true";
 
 if (isHappy) {
-  console.log("I am happy");
-} else {
   console.log("I am not happy");
+} else {
+  console.log("I am happy");
 }
```

Comment:
If you read through this, it now says that if `isHappy` is true, the log "I am not happy". Does that make sense?

TIP: You don't need to change the `if` statement in this example.